assembly/src/assembler/instruction/field_ops.rs

```diff
@@ -28,6 +28,18 @@ pub fn add_imm(span: &mut SpanBuilder, imm: Felt) -> Result<Option<CodeBlock>, A
     }
 }
 
+/// Appends a sequence of operations to subtract an immediate value to the value at the top of the
```

Comment:
nit: "to subtract an immediate value from the value..."

Reply:
done :)